setup.cfg

```diff
@@ -1,4 +1,4 @@
 [pytest]
 python_files = *.py
-addopts = --verbose --ignore setup.py --ignore utide/robustfit.py --doctest-modules --cov-report term-missing --cov .
```

Comment:
> I don't see that the type of functions in this project are very suitable for having examples/doctests, which work only with very small inputs and outputs.

Sold!  I tried to create a small example to paste here and it got so bloated that I gave up.